From 1973e3a432110898d970368b7685ef9c323d5e3f Mon Sep 17 00:00:00 2001
Message-Id: <1973e3a432110898d970368b7685ef9c323d5e3f.1423172493.git.feitong.yi@intel.com>
In-Reply-To: <18bb7324f916bffbc75e07ea0ffd3e6e68e7b6d8.1423172493.git.feitong.yi@intel.com>
References: <18bb7324f916bffbc75e07ea0ffd3e6e68e7b6d8.1423172493.git.feitong.yi@intel.com>
From: Gaurav K Singh <gaurav.k.singh@intel.com>
Date: Tue, 13 Jan 2015 20:26:54 +0530
Subject: [PATCH 12/29] MUST_REBASE [VPG]: drm/i915: Added ADF support for CHV

In order to check for i915.enable_intel_adf kernel param to be enabled,
CHV support needed to be added in the adf sanitize function

MUST_REBASE: Since Google ADF framework is not in upstream and also
we cannot have two display drivers, other one being i915, we need to
work on i915 and ADF convergence path before it can be upstreamed

Issue: GMINL-4443
Change-Id: I5aef6e532ed148aafd3dd6a70bf18df2e757db33
Signed-off-by: Gaurav K Singh <gaurav.k.singh@intel.com>
---
 drivers/gpu/drm/i915/intel_display.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index 70d9c46..1d5e069 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -95,8 +95,8 @@ int intel_sanitize_enable_adf(struct drm_device *dev, int enable_intel_adf)
 	return 0;
 #endif
 
-	/* Current ADF support only for VLV */
-	if (IS_VALLEYVIEW(dev) && IS_GEN7(dev))
+	/* ADF support only for CHV */
+	if (IS_CHERRYVIEW(dev))
 		return 1;
 
 	return 0;
-- 
1.7.9.5

